include/odp/api/abi-default/event_types.h

```diff
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: BSD-3-Clause
  * Copyright (c) 2017-2018 Linaro Limited
- * Copyright (c) 2022-2023 Nokia
+ * Copyright (c) 2022-2024 Nokia
```

Comment:
Copyright years can be updated.

Reply:
I updated copyrights only in the files that actually got changes this year. For this particular file, 2022-2024 is still valid.